### ISSUE TITLE
Shall we add DeFi as a collection type?

### ISSUE THREAD

**Issue by kingthorin** [OWASP/OWASP-VWAD#170]
Shall we add a collection type for DeFi?
https://github.com/tinchoabbate/damn-vulnerable-defi

**Comment by psiinon** [comment]
How many DeFi apps do we have now?
If its just 1 then I'd say no for now...

**Comment by psiinon** [comment]
I couldnt see any others right now (but may have missed some).
I dont think 1 is a collection, not sure how many we'd need, but I think at least 2 :)

**Comment by raulsiles** [comment]
I agree with your previous comments. Once we have "a few", we will need to decide if we create a DeFi category or a Web3 category.

**Comment by kingthorin** [comment]
https://mobile.twitter.com/pashovkrum/status/1647586395823669248
- https://github.com/SunWeb3Sec/DeFiHackLabs
